containers/ws/label-install

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh -eu
+#!/bin/sh
+set -eu
```

Comment:
These kind of changes are good, but unrelated -- would be nice to split into a separate commit. Please prefix them all with "cockpit/ws:".